frontend/package.json

```diff
@@ -80,7 +80,8 @@
     "socket.io-client": "2.0.4",
     "sw-precache": "5.2.0",
     "yup": "0.26.10",
-    "zxcvbn": "4.4.2"
+    "zxcvbn": "4.4.2",
+    "history": "4.7.1"
```

Comment:
Could you add it alphabetically like the others?

Reply:
.